src/jobs/ega/services/permissions.ts

```diff
@@ -44,7 +44,7 @@ export const createRequiredPermissions = async (
   egaClient: EgaClient,
   approvedUser: EgaDacoUser,
   requests: PermissionRequest[],
-) => {
+): Promise<number | undefined> => {
```

Comment:
After reading the descriptio9n and this return type, I don't know what this returns. Why is it a number, why might it be undefined?